### ISSUE TITLE
Resolve accessibility issues on contributing page

### ISSUE THREAD

**Issue by tysongach** [thoughtbot/bitters#206]
Running the [contributing page](https://github.com/thoughtbot/bitters/blob/master/contrib/index.html) through [a11y](https://www.npmjs.com/package/a11y) results in a few issues that we should fix:

- [x] :x: Any ID referred to via an IDREF must be unique in the DOM"

```
#text_field
#text_field
```

- [x] :x: Text elements should have a reasonable contrast ratio:

```
body > .welcome-message > .container > H1 > A
body > .container > P > A
```
